driver-core/src/main/com/mongodb/internal/operation/ClientBulkWriteOperation.java

```diff
@@ -287,11 +408,62 @@ private ExhaustiveClientBulkWriteCommandOkResponse executeBulkWriteCommandAndExh
         }
         List<List<BsonDocument>> cursorExhaustBatches = doWithRetriesDisabledForCommand(retryState, "getMore", () ->
                 exhaustBulkWriteCommandOkResponseCursor(connectionSource, connection, bulkWriteCommandOkResponse));
-        ExhaustiveClientBulkWriteCommandOkResponse exhaustiveBulkWriteCommandOkResponse = new ExhaustiveClientBulkWriteCommandOkResponse(
-                bulkWriteCommandOkResponse, cursorExhaustBatches);
+
```

Comment:
`executeBulkWriteCommandAndExhaustOkResponseAsync` does not have an empty line matching this one in `executeBulkWriteCommandAndExhaustOkResponse`. Let's remove?

```suggestion
```